### PR TITLE
Add ai subnet orch docs

### PR DESCRIPTION
**Pull request by rickstaa**
**What does this pull request do? Explain your changes. (required)**

This commit adds detailed instructions for setting up an orchestrator on the AI Subnet.

**Specific updates (required)**

- adds a `ai-subnet.md` markdown file to the `docs` folder.
- rebases the `ai-video` branch onto the master branch to include https://github.com/livepeer/go-livepeer/pull/2982 and https://github.com/livepeer/go-livepeer/pull/2978 so that the docker image uses CUDA 12.

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [x] Read the [contribution guide](./CONTRIBUTING.md)
- [x] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

**Comment by rickstaa** [comment]
@yondonfu, to ensure the Docker container operates successfully, please rebase your branch onto the master branch. Once completed, I'll rebase my branch onto yours again. I've marked this as a draft as I still need to add additional information for putting it on-chain, such as `recipientAddr`. I'll include this later once my Orchestrator is on-chain. Additionally, to minimize code changes in the docker run command, I've now utilized the `--network host` flag. However, please note that this may introduce security risks which will need to be addressed later.

**Comment by yondonfu** [comment]
> please rebase your branch onto the master branch. 

This is done.

**Comment by rickstaa** [comment]
Now that the CI build the container I will update the documentation. I will also try to include a section that explains how to use the pre-build binaries 👍🏻.

**Comment by rickstaa** [comment]
Replaced by https://github.com/livepeer/go-livepeer-ai/pull/3.

File: doc/ai-subnet.md [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+
+#### Set Ticket Recipient
+
+The first and **recommended method** is to use the `ethOrchAddr` flag to set the Ethereum address of your _Mainnet Transcoding Network_ Orchestrator as the recipient of the AI tickets. This ensures that the AI tickets are redeemed **on-chain** by your _AI Subnet_ Orchestrator, while the AI rewards are sent to your _Mainnet Transcoding Network_ Orchestrator. Follow these steps to set up your _Mainnet AI Subnet_ Orchestrator using this method:
+
+1. Create a new Ethereum account for your _Mainnet AI Subnet_ Orchestrator. It is recommended that you use a separate account from your _Mainnet Transcoding Network_ Orchestrator for security reasons.
+2. Fund the Ethereum account with enough ETH to cover the gas costs of redeeming the AI tickets **on-chain**.
+3. Open port `8936` on your machine and set up port forwarding on your router to make the _Mainnet AI Subnet_ Orchestrator accessible from the internet.
+
```
Comment (JJassonn69) [inline doc/ai-subnet.md:369]: **Orchestrator running in WSL**: If you cannot access your orchestrator in port `8936` even after forwarding port in router and setting up firewall, use the [Netsh interface portproxy](https://learn.microsoft.com/en-us/windows/wsl/networking) guide from *Accessing a WSL 2 distribution from your local area network (LAN)*.

Comment (rickstaa) [reply doc/ai-subnet.md:369]: @JJassonn69 thanks for the update. I will implement your changes in a subsequent pull request.

**Pull request merged**